src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -55,11 +58,14 @@ public class UnsolvedSymbolVisitor extends ModifierVisitor<Void> {
    * The unsolved superclass of this current class file. If there is no unsolved superclass, then
    * the value of this variable is an empty string
    */
-  private @ClassGetSimpleName String superClass = "";
+  private final Map<String, @ClassGetSimpleName String> classAndItsParent = new HashMap<>();
```

Comment:
This change also requires a change to the comment about this field, which is now inaccurate (a map cannot have the value of the empty string).

A comment on a map that is a field should always at a minimum explain what the keys and the values of the map represent.